nightwatch.conf.js

```diff
@@ -0,0 +1,74 @@
+const TRAVIS_JOB_NUMBER = process.env.TRAVIS_JOB_NUMBER;
+
+module.exports = {
+  src_folders: ['test/client'],
+  output_folder: 'reports',
+  custom_commands_path: '',
+  custom_assertions_path: '',
+  page_objects_path: '',
+  globals_path: '',
+
+  selenium: {
+    start_process: true,
+    server_path: './lib/selenium-server-standalone-2.53.0.jar',
+    log_path: '',
+    host: '127.0.0.1',
+    port: 4444,
+    cli_args: {
+      'webdriver.chrome.driver': './lib/chromedriver',
```

Comment:
including binary files in the GitHub repo is a `"nicht nicht"` because they will _inevitably_ change over time and that means anyone cloning the GitHub repo will have to download all the _previous_ versions ... 😮 
I _think_ there's a better solution downloading the latest versions of Selenium & Chromedriver ... see: [learn-nightwatch/nightwatch.conf.js#L123](https://github.com/dwyl/learn-nightwatch/blob/e6a9ebd1e55e0a8a1f4437a102559c5fe54e0d4b/nightwatch.conf.js#L123)
_Feel free to counter-propose_ ...